### DISCUSSION TITLE
Grundlast im Solar-Passthrough Modus berücksichtigen

### DISCUSSION THREAD

**Discussion by Feldsalat** [hoylabs/OpenDTU-OnBattery#681] [Ideas]
Ich verwende meine 24V Speicherbatterie als Spannungsversorgung für diverse Dinge, Fritzbox und andere Netzwerkkomponenten, Raspi und verschiedene Sensoren, eine Logo Kleinsteuerung und anderes. Das erzeugt zusammen eine recht konstante Grundlast von knapp 25 Watt. Im Solar-Passthrough Modus führt das momentan zu einer Entladung der Batterie, da momentan noch wenig von meinen Solarmodulen kommt. Dadurch schaltet der DPL nun zwischen Solar-Passthrough Modus und Batterieladung hin und her. Es wäre daher schön eine einstellbare Grundlast bei der Berechnung der Solar-Passthrough Leistung in Abzug zu bringen. Also von z.B. 100 Watt verfügbarer Solarleistung dann nur 75 Watt für den Inverter zur Einspeisung freigeben. 
Ich denke meine Anwendung ist nicht allzu exotisch und es wäre für andere ebenfalls nützlich.

**Comment by schlimmchen** [comment]
Hm. Für mich klingt das durchaus exotisch...

Und um das umzusetzen bedarf es einer neuen Einstellung, was es grundsätzlich zu vermeiden gilt, insbesondere wenn alle sie irgendwie verstehen und berücksichten müssen, sie aber für nur 2% der Anwender überhaupt interessant ist.

Glaubst du, dass du stattdessen den "Solar Passthrough Losses" Parameter feintunen kannst für deine Anwendung, auch wenn dieser Parameter relativ und nicht absolut in die Berechnung eingeht?

**Comment by Feldsalat** [comment]
Das ist halt gleichzeitig meine USV und spart mir einige Netzteile. 
Der genannte Parameter lässt aktuell nur eine Einstellung bis 10% zu, das hatte ich schon versucht.

**Comment by pelletq** [comment]
Wäre das Problem nicht einfach gelöst. wenn du bei "Angestrebter Netzbezug" 25 oder 30 Watt einträgst?
Oder habe ich da irgendwas nicht verstanden?

**Reply by Feldsalat** [reply]
Meine Grundlast kommt nicht aus dem Netz, sondern direkt aus der Batterie. Die 25W addieren sich also zur Leistungsaufnahme des Inverters und die Batterie entlädt sich ungewollt.
